reference/dtr/2.6/cli/backup.md

```diff
@@ -27,7 +27,8 @@ docker run -i --rm --log-driver none docker/dtr:{{ page.dtr_version }} \
 {% raw %}
 ```bash
 DTR_VERSION=$(docker container inspect $(docker container ps -f \
-  name=dtr-registry -q) | grep -m1 -Po '(?<=DTR_VERSION=)\d.\d.\d'); \
+  name=dtr-registry -q) | grep -m1 DTR_VERSION | \
```

Comment:
@jose-bigio can I please have you test this command on macOS? Not doubting you @omegamormegil - just getting a second pair of eyes. :)

Reply:
Could we use a `--format=....` to get the value? (I assume this version is kept in an env-var on the container/image?)

Reply:
@thaJeztah I was able to get the SHA with the following extraneous command:
`DTR_VERSION=$(docker inspect -f '{{.Image}}' $(docker ps -q -f name=dtr-garant) | cut -f 3 -d '-') && echo $DTR_VERSION`

The version definitely exists in the JSON response to:
```
docker ps --format='{{json .}}' | grep "dtr" | jq . | less
```
Snippet:
```
"Command": "\"/bin/api\"",
  "CreatedAt": "2019-04-02 22:31:44 +0000 UTC",
  "ID": "18d086929ada",
  "Image": "docker/dtr-api:2.6.4"
```

It's my first time playing around with Go template formatting, so need more education. :) cc: @caervs :wave:

Reply:
Ah, hm.. looks like that would still be adding some complication (and requires `jq`, which (although really handy) is not a standard-tool.

Looks like the image does not have a label or env-var set to have the version. Perhaps that's something to consider (but obviously out of scope for this documentation change). e.g.;

```bash
docker build -t lets-pretend-i-am-dtr -<<EOF
FROM busybox
# not needed, but could be convenient in the container itself
ENV DTR_VERSION=2.6.4
LABEL com.docker.product=dtr
LABEL com.docker.product-version=2.6.4
EOF
```

(I'm using reverse-dns notation for the labels; not required, but it prevents them from conflicting with labels that users set, and the `com.docker.` is documented as "reserved for internal use by docker")

With a label in place, we can filter on that label (instead of on the name of the container), so the container could have any name you want:

```bash
docker container create --name any-name-you-like lets-pretend-i-am-dtr:latest
```

And then we can filter containers, and extract the version from the label value;

```bash
docker ps -a \
  --filter label=com.docker.product=dtr \
  --format '{{ .Label "com.docker.product-version" }}'

# prints: 2.6.4
```

Reply:
Going to come back to this later, @thaJeztah , but current thoughts are:
1) Update this command to include extracting the DTR URL
2) Extract the DTR version via this curl request from @cquon :
  `curl -k -X GET --url https://<dtr_url>/basic_info -H "Accept:application/json" -u <admin_username>:<admin_password>`

@cquon @RobbKistler Is this something you can help us tighten up for "advanced usage" of the backup procedure for DTR?

Reply:
@bermudezmt  do you still want me to double check this command?

Is it: 
```
DTR_VERSION=$(docker container inspect $(docker container ps -f \
  name=dtr-registry -q) | grep -m1 DTR_VERSION | \
  sed 's/.*DTR_VERSION=\([0-9].[0-9].[0-9]\)",/\1/'); \
REPLICA_ID=$(docker ps --filter name=dtr-rethinkdb \
  --format "{{ .Names }}" | head -1 | sed 's|.*/||' | sed 's/dtr-rethinkdb-//'); \
read -p 'ucp-url (The UCP URL including domain and port): ' UCP_URL; \
read -p 'ucp-username (The UCP administrator username): ' UCP_ADMIN; \
read -sp 'ucp password: ' UCP_PASSWORD; \
docker run --log-driver none -i --rm \
  --env UCP_PASSWORD=$UCP_PASSWORD \
  docker/dtr:$DTR_VERSION backup \
  --ucp-username $UCP_ADMIN \
  --ucp-url $UCP_URL \
  --ucp-ca "$(curl https://${UCP_URL}/ca)" \
  --existing-replica-id $REPLICA_ID > \
  dtr-metadata-${DTR_VERSION}-backup-$(date +%Y%m%d-%H_%M_%S).tar
```

That's a long command.

Reply:
@jose-bigio the command works for Linux but isn't compatible with macOS. @thaJeztah and I are looking for a cleaner way to extract the DTR version.

Reply:
Honestly we should just backport https://github.com/docker/dhe-deploy/pull/10144